docs/IR.md

```diff
@@ -55,12 +55,13 @@ The following describes the metadata properties of a model graph:
 
 #### Names Within a Graph
 
-Names are organized into separate namespaces, and must be unique within a namespace. The namespaces include the following:
- - `Node`: names that identify specific nodes in the graph, but not necessarily any particular input or output of the node.
- - `Graph`: names that identify graphs in the protobuf.
+Names are organized into separate namespaces. The namespaces include the following:
+ - `Node`: names that identify specific nodes in the graph, but not necessarily any particular input or output of the node. Each node name must be unique.
+ - `Graph`: names that identify graphs in the protobuf. Each graph name must be unique.
```

Comment:
Model scope unique.

Reply:
Actually - do we require graph names to be present? If that's an inlined for loop - it probably doesn't matter that much